pkg/cluster/deployresources_resources.go

```diff
@@ -0,0 +1,553 @@
+package cluster
```

Comment:
I'm OK with adding this file, but it obfuscates any changes made as part of 4.4->4.5 so review is hard.  Please can you rebase and split the relevant commit into two: one which makes a no-op change, the other which has any changes?